superset-frontend/src/explore/components/controls/ColorSchemeControl.jsx

```diff
@@ -74,11 +72,26 @@ export default class ColorSchemeControl extends React.PureComponent {
 
     return (
       <Tooltip id={`${currentScheme.id}-tooltip`} title={currentScheme.label}>
-        <ul className="color-scheme-container" data-test={currentScheme.id}>
+        <ul
+          css={{
```

Comment:
Hi @michael-s-molina. I recall having a small discussion about the css approach VS the currently adopted styled approach. However, I don't recall we taking a decision so I'd assume we should stick with what we have to avoid inconsistencies and bring this conversation to more people to see what they think before going too much in this direction.

Reply:
@geido [Here](https://github.com/apache/superset/pull/12920#discussion_r569471680) is the discussion about the css vs styled 😉. One idea that @rusackas and I were talking about is to create a document of our patterns where everyone can discuss and collaborate.